src/Recaptcha.js

```diff
@@ -21,6 +21,32 @@ export default {
     },
     tabindex: {
       type: String
+    },
+    loadRecaptchaScript: {
+      type: Boolean,
+      default: false
+    },
+    recaptchaScriptId: {
+      type: String,
+      default: '__RECAPTCHA_SCRIPT'
+    },
+    recaptchaHost: {
+      type: String,
+      default: 'www.google.com'
+    }
+  },
+  beforeMount () {
```

Comment:
Just a question. Why prefer to use `beforeMount` instead of `mounted`? Is there any reason? Because if this package is using with SSR. The `beforeMount` hook may be execute at server side.

Reply:
In my project I added the loader at `beforeMount` to make it load as early as possible. I have never worked on Vue SSR, though, so I didn't know about `beforeMount` in SSR.

I googled a bit and found [this](https://ssr.vuejs.org/guide/universal.html#component-lifecycle-hooks):

> Since there are no dynamic updates, of all the lifecycle hooks, only beforeCreate and created will be called during SSR. This means any code inside other lifecycle hooks such as beforeMount or mounted will only be executed on the client.

Should I change it to use `mounted` instead?

Reply:
OK, I think you are right. `beforeMount` will be execute at client side, so you don't need to change it.